src/applications/simple-forms/40-0247/config/form.js

```diff
@@ -90,6 +91,9 @@ const formConfig = {
           uiSchema: vetPersInfoPg.uiSchema,
           schema: vetPersInfoPg.schema,
           pageClass: 'veteran-personal-information',
+          // for this 1st page, default scroll-n-focus is not working
+          scrollAndFocusTarget:
+            'va-segmented-progress-bar[uswds][heading-text][header-level="2"]',
```

Comment:
hm pretty sure we want to do this by default on every page that has this element. not on a form by form basis. because I think there's a lot of forms with this issue.

Reply:
Will fix this for all forms via [this follow-up ticket #661](https://app.zenhub.com/workspaces/vagov-product-team-forms-634853151f5c6000165942bc/issues/gh/department-of-veterans-affairs/va.gov-team-forms/661).